spec/lib/krikri/enrichments/split_coordinates_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+describe Krikri::Enrichments::SplitCoordinates do
+  it_behaves_like 'a field enrichment'
+
+  context 'with place object' do
+    let(:lat) { nil }
+    let(:long) { nil }
+    let(:place) do
+      build(:place, lat: lat, long: long)
+    end
+
+    shared_examples 'latitude and longitude' do |expected_lat, expected_long|
+      it 'assigns latitude' do
+        expect(subject.enrich_value(place).lat).to eq expected_lat
+      end
+      it 'assigns longitude' do
+        expect(subject.enrich_value(place).long).to eq expected_long
+      end      
+    end
+
+    shared_examples 'nonsensical coordinates' do
+      it 'returns empty coordinates' do
+        rv = subject.enrich_value(place)
+        expect(rv.lat).to eq []
+        expect(rv.long).to eq []
+      end
+    end
+
+    context 'with latitude first' do
+      let(:lat) { '40.7127,74.0059' }
+      include_examples 'latitude and longitude', ['40.7127'], ['74.0059']
+    end
+
+    context 'with longitude first' do
+      let(:long) { '40.7127,74.0059' }
+      include_examples 'latitude and longitude', ['74.0059'], ['40.7127']
+    end
+
+    context 'with a space character' do
+      let(:lat) { '40.7127, 74.0059' }
+      include_examples 'latitude and longitude', ['40.7127'], ['74.0059']
+    end
+
+    context 'with a latitude that does not split' do
+      let(:lat) { '40.0' }
+      include_examples 'nonsensical coordinates'
+    end
+
+    context 'with a latitude that splits into too many values' do
+      let(:lat) { '1.0,2.0,3.0' }
+      include_examples 'nonsensical coordinates'
+    end
+
+    context 'with a latitude containing non-decimal characters' do
```

Comment:
I suspect we'll see some data that doesn't include decimal characters, like `51N, 108W`, but I'm happy to punt this until we hit a provider where this applies.